Tests/XcodeGenKitTests/SchemeGeneratorTests.swift

```diff
@@ -43,7 +43,7 @@ class SchemeGeneratorTests: XCTestCase {
     func testSchemes() {
         describe {
 
-            let buildTarget = Scheme.BuildTarget(target: app.name)
+            let buildTarget = Scheme.BuildTarget(target: .init(name: app.name, location: .local))
```

Comment:
Lets have a static helper func on `TargetReference`
```
static func local(_ name: String) -> TargetReference {
  return TargetReference(name: name, location: .local)
}
```
This this becomes `Scheme.BuildTarget(target: .local(app.name))`